test/stringify.js

```diff
@@ -291,4 +291,16 @@ test('stringify()', function (t) {
             return;
         }
     });
+
+    t.test('can use custom encoder for a buffer object', function (st) {
```

Comment:
let's make sure to skip this test if `Buffer` isn't available, since these tests need to pass in browsers and old node versions too. eg `t.test('blah', { skip: condition }, function (st) {`